repairnator/build-rainer/src/main/java/fr/inria/repairnator/BuildRainer.java

```diff
@@ -1,9 +1,51 @@
 package fr.inria.repairnator;
 
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.Map;
+import java.util.HashMap;
+
+import org.java_websocket.client.WebSocketClient;
+import org.java_websocket.drafts.Draft;
+import org.java_websocket.drafts.Draft_6455;
+import org.java_websocket.framing.Framedata;
+import org.java_websocket.handshake.ServerHandshake;
+
+import java.io.IOException;
+import java.io.FileWriter;
 /**
  * This is a websocket, intended for listening to "tdurieux/travis-listener"
  * to fetch most recent builds from Travis in realtime.
  */
-public class BuildRainer {
-    public static void main( String[] args ) {}
-}
+public class BuildRainer extends WebSocketClient {
+
+    public BuildRainer( URI serverURI ) {
+        super( serverURI );
+    }
+
+    @Override
+    public void onMessage( String message ) {
+        System.out.println( "received: " + message);
+    }
+
+    @Override
+    public void onOpen( ServerHandshake handshakedata ) {
+        System.out.println( "opened connection" );
+        send("Hi");
```

Comment:
`BuildRainer` should receive messages from the Travis websocket, right? So this `send("Hi")` seems not necessary. The test case should also test whether `BuildRainer` is able to receive messages.